Add assets for 'BVST-96bbea'

v4nl33 opened the pull request.

paulmultiversx commented:
Hi, @v4nl33 

We have a limitation at only 1 asset / PR so please split it in separate PRs.

Also, don't forget to paste in a comment the signature for the latest commit sha which must be signed with the owner wallet address.

Feel free to use the Sign Message feature from the Utils dApp and paste the signature in a comment for each PR: https://utils.multiversx.com/

Thanks a lot!

v4nl33 commented:
Hi @paulmultiversx,

I removed the devnet token and I added the signature below. Let me know if anything else is needed

0x2c0134966a090e3db8f651fdd20dd464282b4f127e80453edaf2dcb6feae149ff009fe54f7ebc49a95aff0143af44ab25c40a24ff4380f1423e7e8a94df92b03

Have a nice day!
